lib/src/screens/restore/wallet_restore_from_seed_form.dart

```diff
@@ -193,7 +209,9 @@ class WalletRestoreFromSeedFormState extends State<WalletRestoreFromSeedForm> {
                 ),
               ),
             ),
-          if (!isPolyseed && widget.displayBlockHeightSelector)
+          if ((!isPolyseed) &&
+              widget.displayBlockHeightSelector &&
+              (widget.type == WalletType.wownero && !seedTypeController.value.text.contains("14")))
```

Comment:
I believe this is not correct

Reply:
wownero-seed acts like polyseed, and it stores the restore height as well

Reply:
ok, but with this if condition, it has to be a wownero wallet and not a 14 word so that it shows the restore height widget, so if I am on a Monero wallet this condition would return false